apps/ledger-live-mobile/src/components/Web3AppWebview/liveSDKLogic.test.ts

```diff
@@ -21,23 +21,15 @@ describe("prepareSignTransaction", () => {
     // Given
     const parentAccount = createAccount("12");
     const childAccount = createTokenAccount("22", "js:2:ethereum:0x012:");
-    const expectedResult: EvmTransaction = {
+    const expectedResult: Partial<EvmTransaction> = {
```

Comment:
It feels a bit wrong to expect a partial transaction now

Reply:
This is what the prepareSignTransaction returns